### PR TITLE
poprawki tu i owdzie

### PR DESCRIPTION
**Pull request by VladimirKabar**

### CODE REVIEW

File: app/views/static_pages/home/_recent_products.html.erb [hunk 1]
```diff
@@ -4,87 +4,35 @@
...
+                              <%= link_to((image_tag r.image.url), project_path(p)) %>
+                          <% end %>
+                          <span class="spa-sale">Otwórz</span>
+                        </div>
+                        <div class="entry-content woocommerce">
+                          <div class="spa-wrap-line">
+                            <h4 class="entry-title"><%= link_to p.title, project_path(p) %></h4>
+                          </div>
+                        </div>
+                        <p><%= p.description %></p>
```
Comment (maciej-lorens) [inline app/views/static_pages/home/_recent_products.html.erb:26]: move this tag inside `entry-content woocommerce` div, right after `spa-wrap-line` div

File: app/views/static_pages/home/_recent_products.html.erb [hunk 2]
```diff
@@ -4,87 +4,35 @@
...
+                          <% p.resources.each do |r| %>
+                              <%= link_to((image_tag r.image.url), project_path(p)) %>
+                          <% end %>
+                          <span class="spa-sale">Otwórz</span>
+                        </div>
+                        <div class="entry-content woocommerce">
+                          <div class="spa-wrap-line">
+                            <h4 class="entry-title"><%= link_to p.title, project_path(p) %></h4>
+                          </div>
+                        </div>
```
Comment (maciej-lorens) [inline app/views/static_pages/home/_recent_products.html.erb:25]: move `entry-content woocommerce` div outside `entry-thumb spa-hover-effect-2` div. `entry-content woocommerce` div should be on the same level as `entry-thumb spa-hover-effect-2` div, not inside it.

File: db/seeds.rb [hunk 3]
```diff
@@ -7,3 +7,34 @@
...
+          end
+        end
+      end
+    end
+    Resource.create!(title: title,
+                     image: File.new(File.join(Rails.root, "/app/assets/test/#{x+y+1}.jpg")),
+                     description: description,
+                     project: @project)
+  end
+end
```
Comment (maciej-lorens) [inline db/seeds.rb:32]: think about refactoring it, maybe there is better way of creating 4 resources for each project

**Review by maciej-lorens — CHANGES_REQUESTED** [review]

File: db/seeds.rb (hunk 3 above)
Comment (VladimirKabar) [reply db/seeds.rb:32]: done

File: app/views/static_pages/home/_recent_products.html.erb (hunk 1 above)
Comment (VladimirKabar) [reply app/views/static_pages/home/_recent_products.html.erb:26]: done

File: app/views/static_pages/home/_recent_products.html.erb (hunk 2 above)
Comment (VladimirKabar) [reply app/views/static_pages/home/_recent_products.html.erb:25]: done

**Review by maciej-lorens — APPROVED** [review]

**Comment by maciej-lorens** [comment]
approved, can be merged

**Pull request merged**